app/server/lib/ActiveDoc.ts

```diff
@@ -1260,22 +1260,32 @@ export class ActiveDoc extends EventEmitter {
     return this._pyCall('autocomplete', txt, tableId, columnId, rowId, user.toJSON());
   }
 
-  public async getAssistance(docSession: DocSession, userPrompt: Prompt): Promise<Suggestion> {
-    // Making a prompt can leak names of tables and columns.
+  public async getAssistance(docSession: DocSession, request: AssistanceRequest): Promise<AssistanceResponse> {
+    return this.getAssistanceWithOptions(docSession, request);
```

Comment:
Will `getAssistance` and `getAssistanceWithOptions` diverge in the future?

Reply:
`getAssistance` is a method exposed on the webhook API, so values passed to it cannot be trusted. I wanted to pass some trusted options from evaluation code, hence the variant. You can see a similar setup for `applyUserActions` (exposed) forwarding to `_applyUserActionsWithExtendedOptions` (trusted).